1-Authentication/0-sign-in-vanillajs/App/public/index.html

```diff
@@ -60,9 +60,9 @@ <h5 id="welcome-div" class="card-header text-center d-none"></h5>
     <script type="text/javascript" src="./authConfig.js"></script>
     <script type="text/javascript" src="./ui.js"></script>
     <script type="text/javascript" src="./claimUtils.js"></script>
-    <!-- <script type="text/javascript" src="./authRedirect.js"></script> -->
-    <!-- uncomment the above line and comment the line below if you would like to use the redirect flow -->
     <script type="text/javascript" src="./authRedirect.js"></script>
+    <!-- comment the above line and uncomment the line below if you would like to use the popup flow -->
+    <!-- <script type="text/javascript" src="./authPopup.js"></script> -->
```

Comment:
Made this a little clearer, as the tutorial assumes that redirect is your default, not popup.